Akash/html/assign 4/assig2.html

```diff
@@ -24,7 +24,7 @@
     </head>
 
     <body>
-        <h2 style="color: rgb(155, 67, 155);"><center>Jan's Spring Term Afternoon Schedule</center></h2>
+        <h2 style="color: rgb(155, 67, 155);text-align: center;">Jan's Spring Term Afternoon Schedule</h2>
```

Comment:
remove extra lines in this file and a space after colon and before opening bracket there should be a space so add tht and add a space after semicolon